app/uk/gov/hmrc/tariffclassificationfrontend/models/Search.scala

```diff
@@ -27,11 +28,12 @@ case class Search
   commodityCode: Option[String] = None,
   decisionDetails: Option[String] = None,
   status: Option[Set[PseudoCaseStatus]] = None,
+  applicationType: Option[Set[ApplicationType]] = None,
   keywords: Option[Set[String]] = None
 ) {
 
   def isEmpty: Boolean = {
-    // Live rulings only omitted intentionally as it is a post-search filter
+    // Status & Application Type intentionally as it is a post-search filter
```

Comment:
You've omitted the word omitted ;) IF we have comments in, they should make sense